src/index.ts

```diff
@@ -0,0 +1,86 @@
+import { serve } from '@hono/node-server'
+import { Hono } from 'hono'
+import { PrismaClient } from '@prisma/client'
+
+const prisma = new PrismaClient()
+
+const app = new Hono()
+
+app.get('/users', async (c) => {
+  const allUsers = await prisma.users.findMany()
+  return c.json({ data: allUsers, ok: true })
+})
+
+app.get('/users/:id', async (c) => {
+  const id = Number(c.req.param('id'))
```

Comment:
Please add a validation middleware for the id and then use `c.req.valid(‘param’)`